Makefile

```diff
@@ -1,7 +1,6 @@
-langNames.db: language_names/scraped.sql language_names/scraped-sil.sql language_names/manual.sql language_names/variants.sql
+langNames.db: language_names/fixes.tsv language_names/additions.tsv language_names/scraped.tsv language_names/scraped-sil.tsv language_names/turkic_fixes.tsv language_names/turkic_langNames.tsv language_names/variants.tsv language_names/scraper_CLDR.tsv language_names/scraper_SIL.tsv
```

Comment:
Like I mentioned, we don't need multiple versions of the scraped ones. Also, these should go in the same order they were previously and we should not be using any more than previously.

Reply:
Okay I will modify these files

Reply:
Where should I place turkic files?

Reply:
Just don't use the turkic ones for now.

Reply:
Okay I will exclude it